packages/termux-tools/termux-change-repo

```diff
@@ -1,198 +1,65 @@
-
-    elif [ "$1" == "Mirrors by DGUT" ]; then
-        echo "[*] Dongguan University of Technology (CN) mirrors selected"
-        MAIN="https://mirrors.dgut.edu.cn/termux/apt/termux-main"
-        ROOT="https://mirrors.dgut.edu.cn/termux/apt/termux-root"
-        X11="https://mirrors.dgut.edu.cn/termux/apt/termux-x11"
-
-    elif [ "$1" == "Mirrors by HIT" ]; then
-        echo "[*] HIT (CN) mirrors selected"
-        MAIN="https://mirrors.hit.edu.cn/termux/apt/termux-main"
-        ROOT="https://mirrors.hit.edu.cn/termux/apt/termux-root"
-        X11="https://mirrors.hit.edu.cn/termux/apt/termux-x11"
-
-    elif [ "$1" == "Mirrors by NJU" ]; then
-        echo "[*] NJU (CN) mirrors selected"
-        MAIN="https://mirror.nju.edu.cn/termux/apt/termux-main"
-        ROOT="https://mirror.nju.edu.cn/termux/apt/termux-root"
-        X11="https://mirror.nju.edu.cn/termux/apt/termux-x11"
-
-    elif [ "$1" == "Mirrors by Tsinghua University" ]; then
-        echo "[*] Tsinghua's (CN) mirrors selected"
-        MAIN="https://mirrors.tuna.tsinghua.edu.cn/termux/apt/termux-main"
-        ROOT="https://mirrors.tuna.tsinghua.edu.cn/termux/apt/termux-root"
-        X11="https://mirrors.tuna.tsinghua.edu.cn/termux/apt/termux-x11"
-
-    elif [ "$1" == "Mirrors by USTC" ]; then
-        echo "[*] USTC (CN) mirrors selected"
-        MAIN="https://mirrors.ustc.edu.cn/termux/apt/termux-main"
-        ROOT="https://mirrors.ustc.edu.cn/termux/apt/termux-root"
-        X11="https://mirrors.ustc.edu.cn/termux/apt/termux-x11"
-
-    elif [ "$1" == "Mirrors by Alibaba" ]; then
-        echo "[*] Alibaba (CN) mirrors selected"
-        MAIN="https://mirrors.aliyun.com/termux/termux-packages-24"
-        ROOT="https://mirrors.aliyun.com/termux/termux-root-packages-24"
-        X11="https://mirrors.aliyun.com/termux/x11-packages"
+select_repository_group() {
+    if [ "$1" == "Mirrors in Europe" ]; then
+        echo "[*] Mirrors in Europe selected"
+        ln -sf @TERMUX_PREFIX@/etc/termux/mirrors/europe \
+            @TERMUX_PREFIX@/etc/termux/chosen_mirrors
+
+    elif [ "$1" == "Mirrors in North America" ]; then
+        echo "[*] Mirrors in North America selected"
+        ln -sf @TERMUX_PREFIX@/etc/termux/mirrors/north_america \
+            @TERMUX_PREFIX@/etc/termux/chosen_mirrors
+
+    elif [ "$1" == "Mirrors in China" ]; then
+        echo "[*] Mirrors in China selected"
+        ln -sf @TERMUX_PREFIX@/etc/termux/mirrors/china \
+            @TERMUX_PREFIX@/etc/termux/chosen_mirrors
+
+    elif [ "$1" == "Mirrors in Iran" ]; then
+        echo "[*] Mirrors in iran selected"
+        ln -sf @TERMUX_PREFIX@/etc/termux/mirrors/iran \
+            @TERMUX_PREFIX@/etc/termux/chosen_mirrors
+
+    elif [ "$1" == "All mirrors" ]; then
+        echo "[*] All mirrors selected"
+        if [ -L @TERMUX_PREFIX@/etc/termux/chosen_mirrors ]; then
+            unlink @TERMUX_PREFIX@/etc/termux/chosen_mirrors
```

Comment:
Maybe create broken symlink to `@TERMUX_PREFIX@/etc/termux/mirrors/all` and don't show warning in `pkg` script since it was a user decision.